packages/styles/src/themes/pentahoPlus.ts

```diff
@@ -390,41 +390,114 @@ const pentahoPlus = makeTheme((theme) => ({
     HvButton: {
       classes: {
         root: {
-          borderRadius: theme.radii.full,
+          border: "none",
+          borderRadius: 48,
```

Comment:
why move away from our theme token? aren't they the same?

Reply:
We have no token for this `48px` value, I think.